File: Github/codigosDeExemplo-master/codigosDeExemplo-master/WEB_PHP_GIT/sysVendas_crudProduto/connection/FabricaConexao.php
<?php
	class FabricaConexao{
	 static function conectar(){
		$host = "localhost";
		$dbname = "sysVendas";
		$user = "root";
		$password = "";
			
		$conn = new PDO("mysql:host=$host;dbname=$dbname", 
						$user, $password);
								
		return $conn;		
	 }
	}	
?>	

File: Códigos/LPOO_GIT/src/conexaoBD/connection/FabricaConexao.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package conexaoBD.connection;

import java.sql.*;

public class FabricaConexao {
    
   public static Connection getConnection(){
       try{
           String host = "jdbc:mysql://localhost/sysControleAcademico";
           String user = "root";
           String password = "";
           return DriverManager.getConnection(
                   host, user, password);
           
       }catch(SQLException e){
           throw new RuntimeException(e);
       } 
   }
}
